packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Withdraw/BankPicker/template.success.tsx

```diff
@@ -22,19 +22,22 @@ const Top = styled.div`
   align-items: center;
 `
 const AlertWrapper = styled(FlyoutWrapper)`
+  padding-bottom: 1rem;
+  padding-top: 0;
   & > div {
     width: 100% !important;
   }
 `
 
+const noOp = () => {}
```

Comment:
What is the reason for this empty callback function?

Reply:
`onClick` is required down below but it _shouldn't_ be used. Having it here so I don't have to inline it and it does not get recreated every time.